.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Feature request
+about: " Suggest a new feature to enhance Cantera"
+title: ''
+labels: feature-request
+assignees: ''
+
+---
+
+**Abstract**
+
+A brief description of the proposed change.
+
+**Motivation**
+
+Describe the need for the proposed change:
+
+* What problem is it trying to solve?
+* Who is affected by the change?
+* Why is this a good solution?
+
+**Description**
```

Comment:
One thought that comes to mind: this template seems to assume that the requestor has a solution in mind.  There should either be a separate template for somebody opening a request but w/ no idea how to solve it, or a hearty dose of "(if applicable)" in this template.

Reply:
That's a good point -- a feature request doesn't need that much detail on the implementation, at least to start. Updated accordingly.